tests/providers/google/marketing_platform/operators/test_display_video_system.py

```diff
@@ -17,8 +17,8 @@
 import pytest
```

Comment:
In the new design this file is not required. Please ensure that the setup/teardown fixtures are migrated to the DAG itself.